packages/bpk-stylesheets/base.js

```diff
@@ -94,7 +94,7 @@
+eval("__webpack_require__.r(__webpack_exports__);\n/* harmony import */ var _index_scss__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./index.scss */ \"./index.scss\");\n/* harmony import */ var _index_scss__WEBPACK_IMPORTED_MODULE_0___default = /*#__PURE__*/__webpack_require__.n(_index_scss__WEBPACK_IMPORTED_MODULE_0__);\n/*\n * Backpack - Skyscanner's Design System\n *\n * Copyright 2016-2020 Skyscanner Ltd\n *\n * Licensed under the Apache License, Version 2.0 (the \"License\");\n * you may not use this file except in compliance with the License.\n * You may obtain a copy of the License at\n *\n *   http://www.apache.org/licenses/LICENSE-2.0\n *\n * Unless required by applicable law or agreed to in writing, software\n * distributed under the License is distributed on an \"AS IS\" BASIS,\n * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n * See the License for the specific language governing permissions and\n * limitations under the License.\n */\n\n\n(function () {\n  if (typeof document === 'undefined') {\n    return;\n  }\n\n  var classNames = []; // touch support\n\n  classNames.push('ontouchstart' in window || window.DocumentTouch && document instanceof DocumentTouch // eslint-disable-line no-undef\n  ? 'touch-support' : 'no-touch-support'); // add more feature tests here...\n\n  document.documentElement.className += \" \".concat(classNames.map(function (className) {\n    return \"bpk-\".concat(className);\n  }).join(' '));\n})();\n\n//# sourceURL=webpack:///./index.js?");
```

Comment:
This change has come from the updates to indentation and the updated eslint rule in `index.js`

Reply:
Exactly, sorry I forgot to mention this 😅